client/lib/media/actions.js

```diff
@@ -260,7 +260,9 @@ MediaActions.update = function( siteId, item, editMediaFile = false ) {
 				type: 'RECEIVE_MEDIA_ITEM',
 				error: error,
 				siteId: siteId,
-				data: data
+				data: editMediaFile
+					? { ...data, isDirty: true }
```

Comment:
Do we need it, though? We are only doing things if `isDirty` is `true`. Otherwise, nothing happens.